projects/core/src/configurator/commons/facade/configurator-cart.service.ts

```diff
@@ -26,7 +26,7 @@ export class ConfiguratorCartService {
    * Checks if there are no updates for the active cart pending
    * @returns Observable emitting that no changes are pending
```

Comment:
This description is not correct
Change to: Observable emitting that weather changes are pending or not

Reply:
Method will be deleted...